README.md

```diff
@@ -85,25 +85,30 @@ add them to the NEXTCLOUD_AUTOINSTALL_APPS variable in `.env`.
 
 ## Complex setup
 
-In order to achieve a more complex dev environment with different branches of the server, follow the manual steps:
+In order to achieve a more complex dev environment with different branches of the server, follow the following manual steps:
 
-##### 1. Clone the repository
+##### 1. Clone and prepare the repository
+
+1. Run the setup from the "Simple master setup" from above.
+2. For this setup it is nessesary to do the steps that are descripte in the note from the "Simple master setup".
 
 ```
-git clone https://github.com/nextcloud/server.git
+cd workspace/server
```

Comment:
I'm confused. The git clone way worked fine for me a while ago. 
You rewrote the tutorial to call boostrap.sh first and upgrade the setup to "complex" then?

Reply:
I'd agree with @datenangebot that the complex setup is something that is rather building on top of the simple setup (with calling bootstrap.sh). Iirc this section was part of resolving the issues that @camilasan had when setting up stable branches but their bootstrap was already called.

Maybe the term complex setup is a bit confusing as it is still the same base setup that is required.